module/programs/helper.js

```diff
@@ -150,6 +197,827 @@ module.exports = class ProgramsHelper {
     });
   }
 
+   /**
+   * Update program
+   * @method
+   * @name update
+   * @param {String} programId - program id.
+   * @param {Array} data
+   * @param {String} userId
+   * @param {Boolean} checkDate this is true for when its called via API calls
+   * @returns {JSON} - update program.
+   */
+
+  static update(programId, data, userId, checkDate = false,token) {
```

Comment:
@praveenKDass do we require token here?

Reply:
No removing the token @VISHNUDAS-tunerlabs